scripts/qa_crowbarsetup.sh

```diff
@@ -273,11 +273,23 @@ function add_mount()
     local nfssrc="$2"
     local targetdir="$3"
     local zypper_alias="$4"
-    if [ -n "${localreposdir_target}" ]; then
-        [ -n "${bindsrc}" ] && add_bind_mount "${localreposdir_target}/${bindsrc}" "${targetdir}"
+
+    if [ -n "$localreposdir_target" ]; then
+        if [ -z "$bindsrc" ]; then
+            complain 50 "BUG: add_mount() called with empty bindsrc parameter" \
```

Comment:
i know you prefer it like this, but we should do it like [here](https://github.com/SUSE-Cloud/automation/commit/7900657d1b95186f5af6b2b53ba5c5f02ae45298) to be consistent in complaining